docs/demo.rst

```diff
@@ -185,7 +185,7 @@ performed above, we can use the following code:
 
     from __future__ import print_function
 
-    import ga4gh.client.client as client
+    import ga4gh.client as client
```

Comment:
The signature has changed in the latest version. I personally prefer `from ga4gh.client import client`, but the line you've removed is correct for the current development version. 0.3.4 released version uses the syntax you've provided.